pkg/certificate/interfaces/interface.go

```diff
@@ -20,7 +20,7 @@ type Config struct {
 	Organization     []string
 	AltNames         AltNames
 	ValidityDuration time.Duration
-	CABundle         []byte
+	CABundleSecret   string
```

Comment:
Quick question, why did you change this? Is this because of https://github.com/etcd-io/etcd-operator/pull/103/files#diff-1d55d8dd7e081f3e3c7c5ab245f407c05a1b8b49903385124be229947e77a555R85-R89?

(I may be lacking some context)

Reply:
Yes, that is correct. CABundle will be stored in a secret which will be passed by the user. Also, this will close Issue https://github.com/etcd-io/etcd-operator/issues/104.

Reply:
Thanks for the explanation :)